### ISSUE TITLE
The output of validate validate_sbml_model is not complete

### ISSUE THREAD

**Issue by ChristianLieven** [opencobra/cobrapy#702]
#### Problem description

I tried to use `validate_sbml_model` to validate an SBML model. The online SBML validator shows more than 150 errors whereas the output of this function shows no errors.

#### Code Sample

Obtain the model from https://github.com/ChristianLieven/Yarrowia_lipolytica_W29-GEM/tree/fix-sbml-errors/ModelFiles/xml

Then run:

```python
from cobra.io.sbml3 import validate_sbml_model

validate_sbml_model(("Yarrowia_lipolytica_W29-GEM/ModelFiles/xml/iYali4.xml"))
```

#### Actual Output
```
(None,
 {'validator': [],
  'warnings': [],
  'SBML errors': [],
  'other': ["'NoneType' object has no attribute 'startswith'"]})
```

#### Expected Output

Something like this?
```
(None,
 {'validator': [],
  'warnings': [],
  'SBML errors': [ERROR1, ERROR2, ERROR3, etc],
  'other': ["'NoneType' object has no attribute 'startswith'"]})
```

This is the output of the online SBML validator:
<img width="756" alt="screen shot 2018-05-04 at 12 54 18" src="https://user-images.githubusercontent.com/8027271/39624666-72d931cc-4f9a-11e8-9dac-0651095b3b7b.png">

#### Output of `cobra.show_versions()`

<details>

System Information
==================
OS              Darwin
OS-release      16.7.0
Python           3.6.5

Package Versions
================
pip              9.0.3
setuptools      39.1.0
cobra           0.11.3
future          0.16.0
swiglpk          1.4.4
optlang          1.4.1
ruamel.yaml    0.14.12
pandas          0.22.0
numpy           1.14.3
tabulate         0.8.2
python-libsbml  5.16.0
scipy            1.0.1

</details>

**Comment by cdiener** [comment]
Yeah, the internal SBML validator is definitely not a complete one. It rather checks for a small subset of common problems. The validator output is a bit weird though. For instance no. 4 looks like a valid chemical formula and does not contain the bracket the validator complains about.

**Comment by ChristianLieven** [comment]
I sometimes think the traceback points to the metabolite right before or after the one that actually is broken. It's definitely been a pet peeve of mine how cryptic it is sometimes.

**Comment by matthiaskoenig** [comment]
This was fixed in the recent release 0.15.3 (just forgot to close this issue).
Please open a new issue if there are problems with `validate_sbml_model`.
